### DISCUSSION TITLE
Hide left panel in netbox 4.0.7

### DISCUSSION THREAD

**Discussion by mmip-labs** [netbox-community/netbox#16910] [General]
Hello there

How can i collapse left menu bar in Netbox 4.0.7, like in version 3.7.8? There is not pin button in 4.0.7

![2024-07-13_08-06-28](https://github.com/user-attachments/assets/911642c4-b11f-4a97-800a-c94e90598bd9)

**Comment by candlerb** [comment]
You can't currently, but it's being worked on. #16392, #15614.

**Comment by mmip-labs** [comment]
Thank you!
